Backpack/src/main/res/values/backpack.color.xml

```diff
@@ -202,14 +190,46 @@
 
 	<color name="bpkWhiteAlpha80">#8e8e93</color>
 
-	<color name="bpkYellow">#feeb87</color>
-
 	<color name="bpkYellowMuted">#ffebd0</color>
 
 	<color name="bpkLineDark">#48484a</color>
 
 	<color name="bpkLineLight">#cdcdd7</color>
 
+	<color name="bpkMarcommsBerry">#d1435b</color>
```

Comment:
we can't split out colours in the old xml layout (I mean we could put them in a separate file, but it would have no impact on usage)